RockLib.Configuration.AspNetCore/RockLib.Configuration.AspNetCore.csproj

```diff
@@ -41,14 +36,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="RockLib.Configuration" Version="2.5.3" />
+    <PackageReference Include="RockLib.Configuration" Version="3.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='net5.0'">
+  <ItemGroup Condition="'$(TargetFramework)'=='net6.0' or '$(TargetFramework)'=='netcoreapp3.1'">
```

Comment:
Hrm do we need this at all?  https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-6.0.